apps/web/src/components/ui/sidenav-project.tsx

```diff
@@ -47,11 +47,11 @@ export function SidenavProject({ className }: { className?: string }) {
             <span className="truncate">Instellingen</span>
             <ChevronDown
               size={16}
-              className={
-                location.includes('/settings')
-                  ? 'rotate-180 duration-200'
-                  : null
-              }
+              // className={
```

Comment:
Deze klasse wordt niet meer gebruikt? Verwijder dan de commented code

Reply:
Deze zal waarschijnlijk nog wel gebruikt worden, maar de klasse ChevronDown ondersteunt op zichzelf niet 'className'. Ik hield het erin als herinnering dat deze als het kan later hopelijk aangepast kunnen worden, maar voor deze merge zal ik ze weglaten (het breekt verder toch niks.)

Reply:
Dat is niet de error zoals ik het zie
Er staat dat classname niet null aankan, maar alleen string. Dus bij de false kant moet je dan een lege string hebben

Reply:
<ChevronDown
              size={16}
              className={
                location.includes('/settings')
                  ? 'rotate-180 duration-200'
                  : '' ipv null
              }